src/components/Header/Header.tsx

```diff
@@ -18,13 +22,15 @@ export default function Header() {
 
   return (
     <header className="flex flex-row justify-between  items-center text-center w-screen border border-black">
-      <h1 className="text-7xl bg-gray-custom p-5">JSJ</h1>
+      <Link href={"/"} className="text-7xl bg-gray-custom p-5">
+        JSJ
+      </Link>
       <>
         <ul className="flex space-x-6 text-2xl">
           {categories.map((cat) => (
             <li key={cat.name}>
               <Link
-                className="font-extrabold cursor-pointer hover:underline"
+                className={`${pathname === cat.link ? "underline" : ""} font-extrabold cursor-pointer hover:underline`}
```

Comment:
ohh! I learned something new! thanks

Reply:
@judy0ye Same! Tailwind is growing on me